client/src/controllers/scores/scores_controller.js

```diff
@@ -2,58 +2,79 @@
 
 class ScoresController {
 
-	constructor ($scope, Scores, Messanger) {
+	constructor($scope, Scores, Tournament, Messanger) {
 		this.$scope = $scope
 		this.Scores = Scores
+		this.Tournament = Tournament
 		this.Messanger = Messanger
 		this.search = ''
 		this.showDuplicates = false
+		this.showErrors = false
+		this._loading = true
 	}
 
-	$onInit () {
+	$onInit() {
 		let self = this
 		this.load(false)
 		// If the reload event comes from within this client, reload and send the message to every other client
 		// Otherwise just reload
 		this.$scope.$on('reload', () => self.load(true))
 		this.Messanger.on('reload', () => self.load(false), true)
+		Promise.all([this.Tournament.teams()]).then(responses => {

```

Comment:
Just checked it on my computer and it worked. Please tell me what exactly is not working? You need to remember that after not using `Promise.all`, the `then` will not recieve an array, but simply the list of teams, like this:
``this.Tournament.teams().then(teams => {``. This is more accurate then what I wrote yesterday